src/Esfa.Vacancy.Register.Infrastructure/Repositories/FrameworkCodeRepository.cs

```diff
@@ -1,31 +1,39 @@
 ﻿using System.Collections.Generic;
+using System.Data;
 using System.Data.SqlClient;
 using System.Threading.Tasks;
 using Dapper;
 using Esfa.Vacancy.Register.Domain.Repositories;
 using Esfa.Vacancy.Register.Infrastructure.Settings;
+using SFA.DAS.NLog.Logger;
 
 namespace Esfa.Vacancy.Register.Infrastructure.Repositories
 {
     public class FrameworkCodeRepository : IFrameworkCodeRepository
     {
         private readonly IProvideSettings _settings;
+        private readonly ILog _logger;
 
-        private const string GetFrameworkCodesQuery = @"
-            SELECT 
-                CodeName
-            FROM 
-                dbo.ApprenticeshipFramework
-            WHERE 
-                ApprenticeshipFrameworkStatusTypeId = 1";
+        private const string GetFrameworkCodesQuery = "[VACANCY_API].[GetActiveFrameworkCodes]";
```

Comment:
Could this be called GetActiveFrameworksSqlSproc?

Reply:
what about: 
GetActiveStandardCodesStoredProcedure, 
GetActiveFrameworkCodesStoredProcedure